code/__DEFINES/tick.dm

```diff
@@ -1,5 +1,10 @@
-#define TICK_LIMIT_RUNNING 90
-#define TICK_LIMIT_TO_RUN 85
+#define TICK_LIMIT_RUNNING 80
+#define TICK_LIMIT_TO_RUN 70
+#define TICK_LIMIT_MC 70
+#define TICK_LIMIT_MC_INIT_DEFAULT 98
+
+#define TICK_USAGE world.tick_usage //for general usage
+#define TICK_USAGE_REAL world.tick_usage    //to be used where the result isn't checked
 
 #define TICK_CHECK ( world.tick_usage > TICK_LIMIT_RUNNING ? stoplag() : 0 )
 #define CHECK_TICK if(world.tick_usage > TICK_LIMIT_RUNNING)  stoplag()
```

Comment:
I do wonder if this should be changed *now* or later.

Reply:
I opted not to do so because I was concerned it may interfere with the process scheduler.